src/screens/charging-stations/list/ChargingStations.tsx

```diff
@@ -184,6 +185,8 @@ export default class ChargingStations extends BaseAutoRefreshScreen<Props, State
     if (this.siteAreaID) {
       // Go Back
       this.props.navigation.goBack();
+    } else if (this.state.showMap && !Utils.isEmptyArray(this.state.chargingStations)) {
+      this.setState({ showMap: false });
```

Comment:
I think you can remove the line 187 and  just have if/else like on siteAreas but wrapped in the if(this.siteAreaID)